### DISCUSSION TITLE
[Feature Request] Config file

### DISCUSSION THREAD

**Discussion by Gamemassa** [CorgiTaco-MC/Enhanced-Celestials#140] [Ideas]
Would it be possible to make this mod configurable with the options below?

- Option to select which moons can occur in a server/world (e.g. if I only wanted my server to have the Blue and Blood moons, but not the Harvest, etc.)
- Option to change frequency of each moon type
- Option to increase/decrease the effects of each moon type (e.g. higher spawn rate for Blood, etc.)

**Comment by CorgiTaco** [comment]
@Gamemassa this stuff is configurable via datapacks in 1.19, in 1.18 you can find these configs in your config directory

**Comment by Gamemassa** [comment]
I'm not really well-versed with datapacking. Would you mind giving me a quick rundown of how to configure these things?

**Comment by CorgiTaco** [comment]
@Gamemassa https://github.com/CorgiTaco/Enhanced-Celestials/wiki
